### DISCUSSION TITLE
【注意】对接口做大量调试（可能）会导致封号90天

### DISCUSSION THREAD

**Discussion by Z-fly** [lanyeeee/bilibili-manga-downloader#52] [General]
![](https://github.com/user-attachments/assets/6074a0fb-4d3d-4f88-b19e-2de36211e7d0)

**Comment by Z-fly** [comment]
可能导致封禁的原因：大量请求同个接口、大量爬取漫画、在触发风控前提下继续请求接口等

注意：随着风控反爬虫力度提升，被封过号的，解封后 可能会被风控力度针对 就是这个号被系统盯上了 下次就不止90天了估计

**Comment by lanyeeee** [comment]
唉，又是一个没通知到的用户

本来的确是不风控的，因为B漫那边接口变动就变成风控了
看来在解决风控问题之前，得先加个检查更新功能来通知用户(更新/别再使用)

风控的问题已经在着手解决了，目前基本稳定了，只是我没发代码没发版本
~~为了摸清风控的规则我已经封了不下5个号了~~

**Comment by lanyeeee** [comment]
你直接发请求很容易风控的，特别是网页端

**Comment by Z-fly** [comment]
90天 退款估计也没办法了
损失挺大的

**Comment by lanyeeee** [comment]
以前B漫是完全不管的，想下多快就下多快，现在突然就特别严格，比如你的情况，光是调试都可能封号

**Comment by Z-fly** [comment]
拷贝漫画已经看不了大量B漫了，估计B漫也注意到这些了

**Comment by NSun200512** [comment]
噗我这边也因为用了这软件被封了，不过是30天，客服给的理由是“多设备或异地登录”(噗)
另外风控的问题真的基本解决了？期待下个版本（

**Comment by Z-fly** [comment]
多设备或异地登录 就是理由模版 它不会跟你说风控细节的

**Comment by NSun200512** [comment]
是这样的，所以更重要的是看你这边搞好了没有(？)
<s>我还是先注册个小号吧，感觉好麻烦</s>

**Comment by Z-fly** [comment]
> 以前B漫是完全不管的，想下多快就下多快，现在突然就特别严格，比如你的情况，光是调试都可能封号

这个仓库被b站官方开发人员注意了 我有种感觉就是这几天把封禁力度提上去的

**Comment by lanyeeee** [comment]
试试 [v0.9.0](https://github.com/lanyeeee/bilibili-manga-downloader/releases/tag/v0.9.0)

**Comment by lanyeeee** [comment]
v0.9.0也别用了

看起来只有下载付费章节才会封号，我昨天在登录状态下下了一整天免费的章节都没事，如果是这样的话试错成本就太高了

**Reply by Z-fly** [reply]
免费章节下载原图会封号吗

**Reply by lanyeeee** [reply]
免费章节下载原图 v0.9.0不会封，v0.8会封

**Reply by Z-fly** [reply]
免费章节不用登录也能下吧 我觉得可以加个不带cookie下载免费章节的功能
防止以后改接口再被风控

**Reply by lanyeeee** [reply]
没意义，在下付费章节的原图封号的前提下，这些改动只会徒增代码复杂度和工作量

**Reply by Z-fly** [reply]
那这个项目还会弄下去吗

**Reply by Z-fly** [reply]
不过我看你禁漫那个是做了登录和不登录两种下载的

**Reply by lanyeeee** [reply]
> 那这个项目还会弄下去吗

下载付费章节封号的问题已经在着手解决了，我先自己下一天，不封号就发出来

**Comment by lanyeeee** [comment]
试试 [v0.10.0](https://github.com/lanyeeee/bilibili-manga-downloader/releases/tag/v0.10.0)

**Reply by Z-fly** [reply]
拿没被封号过的下载 没有触发风控
封过号的 得解封后再试 但下次万一直接再封90天以上 不敢试了
